src/sinks/kafka.rs

```diff
@@ -210,13 +214,23 @@ impl Sink for KafkaSink {
 
 fn healthcheck(config: KafkaSinkConfig) -> super::Healthcheck {
     let client = config.to_rdkafka().unwrap();
+    let topic = match Template::from(config.topic).render_string(&Event::from("")) {
```

Comment:
Instead of rendering an empty event, it might be better to check `Template::is_dynamic` and then just pass `None` to `fetch_metadata` below if it is.. That way we can still check that we can connect to the cluster.

Reply:
My thinking was that template items like a date stamp could still be rendered and be able to do the full health check. However, I can do what you suggest for the metadata fetch part.

Reply:
Yeah, that's fine with me!